test/contractsv2/UpgradeToPP.test.ts

```diff
@@ -4,36 +4,29 @@
 /* eslint-disable no-plusplus, no-await-in-loop */
```

Comment:
Some things re this file:
* The diff is missleading. I havent added code, just removed. And some very very minor renaming and changes.
* To avoid more complexity in here, leaving this as TODO. This file is named `UpgradeToPP` but there is a `should create rollup type validium & migrate to ALGateway` test. I guess that test doesn't belong here. If so, will move it in a different PR.